README.md

```diff
@@ -109,14 +101,6 @@ Note that for all merges from master or stable, the merge should pick up:
 But never:
 * changes that bumped to a SHA inside config/components
 
-Here's a sample snippet used for a stable -> aardwolf merge:
```

Comment:
Should I attempt to update this language? I'm not entirely sure I'm qualified to do that.